src/core/i18n/utils.js

```diff
@@ -191,10 +191,10 @@ export function makeI18n(i18nData, lang, _Jed = Jed) {
 
   // This adds the correct moment locale for the active locale so we can get
   // localised dates, times, etc.
-  if (i18n.options && typeof i18n.options._momentDefineLocale === 'function') {
-    i18n.options._momentDefineLocale();
-    moment.locale(makeMomentLocale(i18n.lang));
-  }
+  // if (i18n.options && typeof i18n.options._momentDefineLocale === 'function') {
```

Comment:
Moment related again, the moment data was removed so this wouldn't work as it did before.